### PR TITLE
Sept marketplace release

### PR DESCRIPTION
**Pull request by tbaka**
Adds deployment docs for HashiCorp Nomad Cluster and HashiCorp Nomad Clients Cluster

### CODE REVIEW

File: docs/products/tools/marketplace/guides/hashicorp-nomad-clients-cluster/index.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+keywords: ['HashiCorp','Nomad','Scheduling', 'orchestration']
+tags: ["marketplace", "linode platform", "cloud manager"]
+published: 2023-09-11
+modified_by:
+  name: Linode
+title: "Deploy HashiCorp Nomad Clients Cluster through the Linode Marketplace"
+authors: ["Linode"]
+---
+
+[HashiCorp Nomad](https://www.nomadproject.io/) is a workload scheduler and orchestrator that allows you to deploy and manage containerized or non-containerized, legacy applications using a single, unified workflow. Nomad uses bin packaging for efficient job scheduling and resource optimization, as well as enables developers to use declarative infrastructure-as-code for their deployments.
```
Comment (andystevensname) [inline docs/products/tools/marketplace/guides/hashicorp-nomad-clients-cluster/index.md:12]: The opening paragraph for this and the other OCC should address their differences.

File: docs/products/tools/marketplace/guides/hashicorp-nomad-clients-cluster/index.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+{{< content "marketplace-special-character-limitations-shortguide">}}
+
+### Nomad Client Options
+
+- **Linode API Token** *(required)*: Your API token is used to deploy additional Compute Instances as part of this cluster. At a minimum, this token must have Read/Write access to *Linodes*. If you do not yet have an API token, see [Get an API Access Token](/docs/products/tools/api/guides/manage-api-tokens/) to create one.
+
+- **Limited sudo user** *(required)*: A limited user account with sudo access is created as part of this cluster deployment. Enter your preferred username for this limited user. Please note that the password is automatically created.
+
+- **consul_nomad_autojoin_token generated by Nomad Server OCC** *(required)*: This token is generated by the [HashiCorp Nomad Cluster](/docs/products/tools/marketplace/guides/hashicorp-nomad-cluster/) deployment and can be found in the `~/.deployment_secrets.txt` file created in your original cluster.
```
Comment (andystevensname) [inline docs/products/tools/marketplace/guides/hashicorp-nomad-clients-cluster/index.md:44]: I might be missing it, but we should mention this somewhere earlier. Is here is the first place that mentions the other OCC?

**Review by andystevensname — APPROVED** [review]

File: docs/products/tools/marketplace/guides/hashicorp-nomad-clients-cluster/index.md (hunk 1 above)
Comment (jddocs) [reply docs/products/tools/marketplace/guides/hashicorp-nomad-clients-cluster/index.md]: Edited!

File: docs/products/tools/marketplace/guides/hashicorp-nomad-clients-cluster/index.md (hunk 2 above)
Comment (jddocs) [reply docs/products/tools/marketplace/guides/hashicorp-nomad-clients-cluster/index.md:44]: We did mention it earlier, but I called it out a little more obviously when editing the opening paragraph.

**Pull request merged**